src/library/dao/playlistdao.cpp

```diff
@@ -863,7 +863,9 @@ void PlaylistDAO::shuffleTracks(const int playlistId, const QList<int>& position
     qsrand(seed);
     QHash<int,TrackId> trackPositionIds = allIds;
     QList<int> newPositions = positions;
+    QList<QPair<int, int> > playListTracksPositionRowIds;
```

Comment:
When talking about "track positions" in the PlaylistDAO the prefix "playList" is redundant